README.md

```diff
@@ -36,6 +36,12 @@ There are two external libraries needed for HDF5 support in hellbender:
 
 For more information about HDF:  https://www.hdfgroup.org/
 
+Developer note:
+
+The gradle build will handle the Java dependency.  If IntelliJ is configured correctly, it will
```

Comment:
"The gradle build will handle the Java dependency" is not explicit enough.  Instructions need to say explicitly that gradle will automatically download and install both hdf libraries.